README.md

```diff
@@ -1,44 +1,251 @@
+...
+...
+✅  Finished templating : infrastructure/terraform/modules/vpc/versions.tf
+⬆  Done Rendering - committing repositories to version control.
+✅  Repository created: github.com/dtoki-test-org/infrastructure
+✅  Repository created: github.com/dtoki-test-org/backend-service
+✅  Repository created: github.com/dtoki-test-org/frontend
+✅  Done - run zero apply to create any required infrastructure or execute any other remote commands to prepare your environments.
+
+
+```
+
+### zero apply
+
+The `zero apply` command takes the templated modules generated based on your input and spins up a scalable & performant infrastructure for you!
+
+```shell
+$ zero apply
+
+# Sample Output
+Choose the environments to apply. This will create infrastructure, CI pipelines, etc.
+At this point, real things will be generated that may cost money!
+Only a single environment may be suitable for an initial test, but for a real system we suggest setting up both staging and production environments.
+✔ Production
+🎉  Bootstrapping project zero-init. Please use the zero-project.yml file to modify the project as needed.
+Cloud provider: AWS
+Runtime platform: Kubernetes
+Infrastructure executor: Terraform
+
+...
+...
+
+
+✅  Done.
+Your projects and infrastructure have been successfully created.  Here are some useful links and commands to get you started:
+zero-aws-eks-stack:
+- Repository URL: github.com/dtoki-test-org/infrastructure-inf
+- To see your kubernetes clusters, run: 'kubectl config get-contexts'
+- To switch to a cluster, use the following commands:
+- for production use: kubectl config use-context arn:aws:eks:us-west-2:514226198018:cluster/dtoki-inf-production-us-west-2
+
+- To inspect the selected cluster, run 'kubectl get node,service,deployment,pods'
+zero-deployable-react-frontend:
+- Repository URL: github.com/dtoki-test-org/frontend-inf
+- Deployment Pipeline URL: https://app.circleci.com/pipelines/github/dtoki-test-org/frontend-inf
+- Production Landing Page: app-dtoki-inf-prod.commitzero.com
+
+zero-deployable-backend:
+- Repository URL: github.com/dtoki-test-org/backend-service-inf
+- Deployment Pipeline URL: https://app.circleci.com/pipelines/github/dtoki-test-org/backend-service-inf
+- Production API: api-dtoki-inf-prod.commitzero.com
+```
+
+***Your stack is now up and running, follow the links your terminal to visit your application 🎉*** 
+
+
+## Zeros Default Stack
+
+![systerm-architecture](https://raw.githubusercontent.com/commitdev/zero-aws-eks-stack/master/templates/docs/architecture-overview.svg)
+
+If you would like to learn more about the zero-aws-eks stack, you can read more about it [here](https://github.com/commitdev/zero-aws-eks-stack/blob/master/README.md)
+___
+
+## Contributing to Zero 
+
+Although zero is still in early development, we welcome collaboration; you can join the [#product-zero] slack channel and follow our [project board]. If you'd like to work a specific ticket / issue, reach out to the team on our slack channel.
```

Comment:
They should create issues in our github repo, don't think we will open our slack to community, in the future we could setup gitter/discord for support type of things. 

we'll need to define some process to accept the issue (or reject saying why we don't think it fits the project)
then categorize them as `feature-request` / `bug` / `enhancements` / `documentations` ...etc

and for minor features / bugs we would either just accept pull-requests.

Reply:
right! I wrote this in the context of the internal alpha release but you're right. a better flow would be to have people create issues on the github repo and we'd have a flow to go over opened issues and tag them etc.. 👍🏽